### PR TITLE
hooli-style-popup

### PR DESCRIPTION
**Pull request by LGeorge007**
### hooli-style-popup

[Demo](https://lgeorge007.github.io/hooli-style-popup/)
[Code base](https://github.com/LGeorge007/hooli-style-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by lenadgit — CHANGES_REQUESTED** [review]
Could you please format your CSS-file by auto-formatting?

File: submissions/LGeorge007/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav-panel">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" class="mail">Mail</a></li>
+                <li class="nav-item"><a href="#" class="img">Images</a></li>
+                <li class="nav-item">
+                    <div class="popup-wrapper">
+                        <input type="checkbox" id="popup-button" class="popup-input">
```
Comment (lenadgit) [inline submissions/LGeorge007/hooli-style-popup/index.html:16]: `id = "popup-button"` - this selector was added what for?  could you please explain?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/LGeorge007/hooli-style-popup/index.html (hunk 1 above)
Comment (LGeorge007) [reply submissions/LGeorge007/hooli-style-popup/index.html:16]: I used input type=checkbox with id = "popup-button".
And use id for binding checkbox and its label.

Comment (LGeorge007) [reply submissions/LGeorge007/hooli-style-popup/index.html]: And yes. I have not used the following construction:
<label>
          <input type = "checkbox">
</label>

I used the standard spelling:
<input type = "checkbox" id = "popup-button">
<label for = "popup-button">.

To hide the popup-input, I use -- appearance: none;

**Comment by LGeorge007** [comment]
> Could you please format your CSS-file by auto-formatting?

Sorry. I will not do this anymore.
Fixed it.

**Review by lenadgit — APPROVED** [review]

**Pull request merged**